src/redux/actions/storeLocation.actions.js

```diff
@@ -1,16 +1,29 @@
 export const STORE_LOCATION_ACTIONS = {
   GET_STORES_BY_PRODUCT_IDS: "GET_STORES_BY_PRODUCT_IDS",
-  SEND_STORES_BY_PRODUCT_IDS: "SEND_STORES_BY_PRODUCT_IDS"
-}
+  SEND_STORES_BY_PRODUCT_IDS: "SEND_STORES_BY_PRODUCT_IDS",
+  GET_USER_CURRENT_LOCATION: "GET_USER_CURRENT_LOCATION",
+  SET_USER_CURRENT_LOCATION: "SET_USER_CURRENT_LOCATION",
+};
 
-export const getStoresByProductIDs = ids => ({
+export const getStoresByProductIDs = payload => ({
```

Comment:
thanks for the review... I will take care of the fix except maybe this one... rationale is:
... the payload is no longer a simple list of ids. It actually contains the current latLng as well ...  with shape payload = {ids: [id1, id2,....], latLng: {lat: 999, lng: 0000}}.